recipes/tensorflow-lite/all/conandata.yml

```
@@ -2,11 +2,29 @@ sources:
   "2.6.0":
     url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.6.0.tar.gz"
     sha256: "41b32eeaddcbc02b0583660bcf508469550e4cd0f86b22d2abe72dfebeacde0f"
+  "2.6.2":
+    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.6.2.tar.gz"
+    sha256: "e68c1d346fc3d529653530ca346b2c62f5b31bd4fcca7ffc9c65bb39ab2f6ed3"
+  "2.9.1":
+    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.9.1.tar.gz"
+    sha256: "6eaf86ead73e23988fe192da1db68f4d3828bcdd0f3a9dc195935e339c95dbdc"
 patches:
-  "2.6.0":
-    - patch_file: "patches/local_find_packages.patch"
+  "2.9.1":
+    - patch_file: "patches/msvc_fixes.patch"
+      patch_description: "Fix Windows build: Apply fixes for compatibility with MSVC from master"
+      patch_type: "backport"
```

Since this is a backpor, its best to link so we can see which version it was added too

```suggestion
      patch_type: "backport"
      patch_source: "https://github.com/tensorflow/tensorflow/commit/79be9efa5190a4a9c828369ba358e6f9e59fbbe5"
```